zebra-state/src/service/finalized_state/disk_format/scan.rs

```diff
@@ -43,6 +43,12 @@ impl From<&[u8; 32]> for SaplingScannedResult {
     }
 }
 
+impl From<SaplingScannedResult> for [u8; 32] {
```

Comment:
This is going to conflict with:
https://github.com/ZcashFoundation/zebra/pull/8075/commits/ea3ca722b2cf71261b2c735211e2f8242fe4d076

Since there's two different ways to turn a [u8; 32] into a transaction ID, I think it would be better to use explicit methods here. (Which is what that PR does.)

Reply:
Makes sense. Thanks for pointing that out. I'm closing this PR without merging.